spec/models/book_spec.rb

```diff
@@ -19,6 +19,47 @@
     it {should have_many :reviews}
   end
 
+  describe "class methods" do
+    describe "when destroying a book" do
+      before(:each) do
+        @book_1 = Book.create!(title: "The Frozen Deep", page_count: 106, year_published: 1874, thumbnail: "https://images.gr-assets.com/books/1328728986l/1009218.jpg")
+        @book_2 = Book.create!(title: "To Kill a Mockingbird", page_count: 281, year_published: 1960, thumbnail: "https://upload.wikimedia.org/wikipedia/en/7/79/To_Kill_a_Mockingbird.JPG")
+        @author_1 = @book_1.authors.create!(name: "Wilkie Collins")
```

Comment:
As above, authors Ivar's not used in this test.

Reply:
Actually these ones are used, because I do assertions on BookAuthor.
```ruby
all_book_ids_in_book_authors = BookAuthor.pluck(:book_id)

expect(all_book_ids_in_book_authors).to_not include(@book_1.id)
expect(all_book_ids_in_book_authors).to include(@book_2.id)
```